test/e2e/features/basic.feature

```diff
@@ -30,17 +30,19 @@ Feature: Basic test
         Then executing "git branch -M main" succeeds
         Then executing "git push -u origin main" succeeds
         Then executing "cd .." succeeds
+        And executing "oc apply -f secrets" succeeds
 
-    Scenario: Bringing the deployment infrastructure up
+    Scenario: Bringing the deployment infrastructure up and create webhook
         Given gitops repository is created
         When executing "kam bootstrap --service-repo-url $SERVICE_REPO_URL --gitops-repo-url $GITOPS_REPO_URL --image-repo $IMAGE_REPO --dockercfgjson $DOCKERCONFIGJSON_PATH --git-host-access-token $GIT_ACCESS_TOKEN --output bootstrap --overwrite" succeeds
+        Then executing "oc apply -f secrets" succeeds
```

Comment:
@varshab1210 The secret resources have the namespace as cicd. How is the cicd namespace being created?

Reply:
There was a misunderstanding referring to the day1 doc. I have made required changes to the pr. Please have a look. Also we need a doc updation so that we are clear with our steps.